terraform/performance/main.tf

```diff
@@ -109,6 +109,14 @@ resource "null_resource" "validator_linux" {
   }
   provisioner "remote-exec" {
     inline = [
+      #mock server dependencies getting transfered.
+      # todo make this optional with a env var.
+      "git clone https://github.com/aws/amazon-cloudwatch-agent-test.git",
```

Comment:
Look at how our other terraform scripts do this. They all use a `github_test_repo` variable.

```
git clone --branch ${var.github_test_repo_branch} ${var.github_test_repo}
```